recipes/servicex/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "servicex" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/servicex-{{ version }}.tar.gz
+  sha256: 10fe7aa45ade9ba978563e20cf51f80aa149139423fd0b0766cd7e72e06e1f74
+
+build:
+  skip: true  # [py<38]
```

Comment:
This is a `noarch` package in reality, but I believe that to use to use things like `# [py<310]` (I forget the technical term that conda-forge uses here) you can't use them with `noarch`.

Reply:
Just remove the selectors, installing those packages in py>39/py310 is not a problem.